float32 roll_rate
float32 pitch_rate
float32 yaw_rate
float32 thrust
int16 kill